intermediate_html_css/intermediate_css_concepts/advanced_selectors.md

```diff
@@ -52,12 +52,12 @@ main div {
 But what if we wanted to be more specific and <span id="childvdesc-knowledge-check">select *only* the `child` or `grand-child` divs?</span> That's where the child combinator `>` comes in handy. Unlike the descendant combinator, it will only select direct children.
 
 ```css
-/* This rule will only select divs with a class of child */
+/* This rule will only select divs with the class child */
 main > div {
   /* Our cool CSS */
 }
 
-/* This rule will only select divs with a class of grand-child */
+/* This rule will only select divs with the class grand-child */
```

Comment:
```suggestion
/* This rule will only select the divs with the class grand-child */
```

Reply:
Right, I missed that. Thanks for pointing it out! LGTM!